### ISSUE TITLE
Expose more info

### ISSUE THREAD

**Issue by pedrofracassi** [beastwilson/nodeyourmeme#1]
It would be nice to have some more information on the meme object, like `url` and `image`

(btw, thanks for the API, I've been looking for one for a long time. Implementing it right now on my [discord bot](https://github.com/pedrofracassi/deku))

**Comment by pedrofracassi** [comment]
#2 should get is working, but probably has bugs. I'm pretty sure that a meme without an image would break it.

**Comment by ArcticZeroo** [comment]
>should get it working, but probably has bugs

Did you not test your PR? You need to test it before we can reasonably accept it. And it can't just fail if there is no image, instead the image property should be `null`.
